src/components/AddItem.js

```diff
@@ -15,9 +15,14 @@ const AddItem = () => {
       });
       alert(`${name} has been successfully added to your list.`);
       setName('');
+      setFrequency(7);
```

Comment:
Resets the default value after form submit.

Reply:
Great catch! Thanks!